src/liboslexec/oslexec_pvt.h

```diff
@@ -32,6 +33,13 @@
 # include <regex>
 #endif
 
+#if BOOST_VERSION >= 105600
```

Comment:
Since you went to the trouble of writing your own - why bring boost into this at all ?

Reply:
Well, I figured it would let us jettison the custom code once our boost floor raises.